x-pack/plugins/index_lifecycle_management/__jest__/client_integration/helpers/types.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Phases } from '../../../common/types';
+
+export type Phase = keyof Phases;
+
+export type TestSubjects =
```

Comment:
@sebelga What are your thoughts on this? I don't think this type helps with correctness, since the test will fail if the wrong test subject is used. I understand it offers nice type hints as you're typing in the data test subject, but I'm personally not a big fan of using types this way.

Reply:
I don't think we can leave it to the dev decision. It won't affect test robustness as it will fail anyway. This is supposed to be an enhancement on the DX so I think it is OK to remove it if it creates more friction than helps 😊

Reply:
[EDIT] I meant: "_I think we can leave it_" 😊